packages/ui/src/components/Visualization/Custom/Group/CustomGroup.scss

```diff
@@ -82,19 +77,19 @@
   }
 
   &:hover {
-    --custom-group--BorderColor: var(--pf-v5-global--primary-color--light-100);
+    --custom-group--BorderColor: #003c70;
   }
 
   &--selected {
-    --custom-group--BorderColor: var(--pf-v5-global--palette--blue-200);
-    --custom-group--Background: var(--pf-v5-global--palette--blue-50);
+    --custom-group--BorderColor: #003c70;
+    --custom-group--Background: #f3f8fc;
```

Comment:
sounds counter-intuitive to not use the Patterfly defaults. It will diverge from the other red Hat products.


Also by not using the var, I fear that it will prevent to use light/dark theme built-in functionalities

Reply:
thanks, I updated it back to the pf colors then